subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -114,7 +114,9 @@ static int download_client_callback(const struct download_client_evt *event)
 			download_client_disconnect(&dlc);
 			LOG_ERR("Download client error");
 			err = dfu_target_done(false);
-			if (err != 0) {
+			if (err != -EACCES) {
```

Comment:
A typo slipped in here, should be `(err == -EACCES)`

Reply:
No worries, I should have checked it myself. I just copy pasted blindly ^^